### DISCUSSION TITLE
primeng 19, angular 19, datepicker touchUI does not work

### DISCUSSION THREAD

**Discussion by leuyvo** [primefaces/community#3222] [PrimeNG] [answered]
primeng: 19.0.0-rc.1,
angular: 19,
datepicker: touchUI does not work

**Comment by Paviloma** [comment]
Any workaround ?

**Reply by Paviloma** [reply, marked as answer]
Workaround : 
Can't wait for official fix, so forked the componant for local use
- Get the datepicker componant from github
- Create css class 
`.p-datepicker-touch-ui {
    position: fixed !important;
    top: 50% !important;
    left: 50% !important;
    min-width: 80vw !important;
    transform: translate(-50%, -50%);
}`
- Add it to the #contentWrapper div
